### PR TITLE
[ADD] web_widget_json_graph: migration to 12.0 T#25074

### PR DESCRIPTION
**Pull request by frahikLV**

### CODE REVIEW

**Comment by frahikLV** [comment]
@keylor2906 can you review this?

File: web_widget_json_graph/static/src/js/web_widget_json_graph.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            }
+            /*jsl:end*/
+        },
+        _destroy: function () {
+            return this._super();
+        },
+    });
+    field_registry.add('json_graph', JSONGraphWidget);
+
+});
```
Comment (keylor2906) [inline web_widget_json_graph/static/src/js/web_widget_json_graph.js]: Missing new line at EOF

File: web_widget_json_graph/static/src/scss/web_widget_json_graph.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    .oe_list_content>tbody>tr>td.oe_list_field_cell {
+        white-space: normal;
+    }
+
+    svg.nvd3-svg {
+        height: 500px;
+        width: 1000px;
+    }
+}
```
Comment (keylor2906) [inline web_widget_json_graph/static/src/scss/web_widget_json_graph.scss]: Missing new line at EOF

File: web_widget_json_graph/static/data.tsv [hunk 3]
```diff
@@ -0,0 +1,1281 @@
...
+7-May-07	103.92
+4-May-07	100.81
+3-May-07	100.40
+2-May-07	100.39
+1-May-07	99.47
+30-Apr-07	99.80
+27-Apr-07	99.92
+26-Apr-07	98.84
+25-Apr-07	95.35
+24-Apr-07	93.24
```
Comment (keylor2906) [inline web_widget_json_graph/static/data.tsv]: Missing new line at EOF

File: web_widget_json_graph/views/templates.xml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
```
Comment (keylor2906) [inline web_widget_json_graph/views/templates.xml]: This must be odoo tags, not openerp

File: web_widget_json_graph/views/templates.xml [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+    <data>
```
Comment (keylor2906) [inline web_widget_json_graph/views/templates.xml]: Remove this old data tags

**Review by keylor2906 — CHANGES_REQUESTED** [review]

**Comment by frahikLV** [comment]
@keylor2906 done 👍

File: web_widget_json_graph/views/templates.xml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <template id="assets_backend" name="web_widget_json_graph assets" inherit_id="web.assets_backend">
+        <xpath expr="." position="inside">
+            <script type="text/javascript" src="/web_widget_json_graph/static/src/js/web_widget_json_graph.js"></script>
+            <link rel="stylesheet" type="text/less" href="/web_widget_json_graph/static/src/less/web_widget_json_graph.less"/>
```
Comment (keylor2906) [inline web_widget_json_graph/views/templates.xml]: @frahikLV Please change this link, the less file was changed for a scss

**Review by keylor2906 — CHANGES_REQUESTED** [review]

**Comment by frahikLV** [comment]
@keylor2906 Done 👍

**Comment by keylor2906** [comment]
@frahikLV please create a dummy instance with the forecast module to functionally test it

**Comment by frahikLV** [comment]
@tomeyro This MR is required by the module, it is pending to merge, its functionality can be seen in the video https://youtu.be/zx7G9S-DUS8

c/c @keylor2906

**Comment by keylor2906** [comment]
@tomeyro It LGTM

**Pull request merged**